packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -43,9 +43,11 @@ void main() {
         ),
       );
 
+      // Not a round number because of floating point error from adding
+      // multiple heights, but should be negligible.
       expect(
         tester.getSize(find.byType(CupertinoTextField)),
-        const Size(200, 29), // 29 is the height of the default font + padding etc.
+        const Size(200, 29.000000476837158), // 29 is the height of the default font + padding etc.
```

Comment:
if this is just floating point error, use a matcher that allows for error.

However, that seems like a VERY LARGE floating point error (literally 8+ orders of magnitude!). What is going on here? We shouldn't have to have that big an error.

Reply:
(same below)

Reply:
This is due to line-height rounding in LibTxt. Will move forwards with this and update the test when rounding is removed in LibTxt (along with all the other tests that may have to be updated)